templates/ra_rdsh_private_autoscale_elb.element

```diff
@@ -122,15 +122,16 @@
             "Type" : "Custom::AmiIdLookup",
             "Properties" :
             {
-                "ServiceToken" :
+                "ServiceToken" :				
```

Comment:
Getting closer, but there shouldn't be extra whitespace at the end of this line, either.